src/app/plugins/addroles.plugin.ts

```diff
@@ -69,12 +70,53 @@ export default class AddRolesPlugin extends Plugin {
       })
     );
 
+    // check for alumni or grad student role, and send a random welcome
+    filteredRoles.forEach((e) => {
+      const roleName = e.name.toLowerCase();
+      const alumniRole = Constants.Roles.Alumni.toLowerCase();
+      const gradStudRole = Constants.Roles.GradStudent.toLowerCase();
+
+      const alumniChannel = this.container.guildService.getChannel(Constants.Channels.Public.AlumniLounge) as TextChannel;
+      const gradStudChannel = this.container.guildService.getChannel(Constants.Channels.Public.GradCafe) as TextChannel;
+
+      if (roleName === alumniRole) {
+        alumniChannel.send(this._getRandomWelcome(member.toString(), alumniRole));
+      }
+
+      if (roleName === gradStudRole) {
+        gradStudChannel.send(this._getRandomWelcome(member.toString(), gradStudRole));
+      } 
+    });
+
     if (filteredRoles.length <= 0) {
       message.reply('Nothing was added successfully.');
     } else {
       message.reply(`Successfully added: ${filteredRoles.map((r) => r.name).join(', ')}`);
     }
   }
+
+  private _getRandomWelcome(user : string, role: string) : string {
+    const alumniWelcomes = [
+      `Congratulations ${user} on all your hard work!`, 
+      `You made it, ${user}! Congratulations!`, 
+      `Congratulations and BRAVO, ${user}`, 
+      `This calls for celebrating! Congratulations ${user}!`,
+      `You did it! Congrats ${user}!`,
+      `Caps off to you, Graduate! Well done, ${user}!`,
+    ];
+
+    const gradStudWelcomes = [
+      `Good luck enduring a few more years of hell ${user}!`,
+      `Hope grad school doesn't take a tool on ya ${user}`,
+      `Welcome ${user}, you are now a Grad Knight!`,
+    ];
```

Comment:
Move these up to be a field in the plugin class
so like
```ts
private static readonly alumniWelcomes = [...]
private static readonly gradStudentWelcomes = [...]
```
right underneath the
```ts
 public static readonly BLACKLISTED_ROLES: string[] = ['suspended'];
```

Reply:
I was actually wondering how you set that up globally. I don't know what variable to replace with `user`.
I know there's a name for this in JS, but I can't find it anywhere

Reply:
Ohh yeah forgot about that, you can make it a function
```ts
const gradStudentWelcomes = (user) => {
    return [
         `Good luck enduring a few more years of hell ${user}!`,
         `Hope grad school doesn't take a tool on ya ${user}`,
         `Welcome ${user}, you are now a Grad Knight!`,
    ];
};
```

And I forgot we have [this function](https://github.com/cs-discord-at-ucf/lion/blob/381d470f0390c7009ff8bea20c99e10edf9b1703/src/common/utils.ts#L1) for getting a random element out of an array.

Then you can use this to get a random sentence
```ts
return getRandom(gradStudWelcomes(user));
```

Reply:
It would probably make more sense for the new function `gradStudentWelcomes = (user) => ...` to use the `getRandom` and just return one sentence, rather than returning an array and choosing a random one outside of it

Reply:
Is it a better idea to make it `private static` and make it somewhat like the `_getRandomWelcome` function?

```js
private static gradStudentWelcomes = (user) => {
...
}
```